DiscordBot/Program.cs

```diff
@@ -1,21 +1,29 @@
 ﻿using System.Threading.Tasks;
+using DiscordBot.Commands;
 using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.Interactivity;
 
 namespace DiscordBot
 {
     internal class Program
     {
+        private static CommandsNextModule commands;
+        private static InteractivityModule interactivity;
+
         private static void Main(string[] args)
         {
-            MainAsync().GetAwaiter().GetResult();
+            MainAsync(args).ConfigureAwait(false).GetAwaiter().GetResult();
         }
 
-        private static async Task MainAsync()
+        private static async Task MainAsync(string[] args)
```

Comment:
Are these arguments used? 🙂

Reply:
Not currently used but was in the example code for the older documentation that we've now switched to.

Reply:
If it's not used then I'd remove it. You can always add it later if you end up needing it 🙂